resources/dicts/thoughts/alive/alive_outside/rogue.json

```diff
@@ -1,61 +1,21 @@
 [
     {
-        "id": "bloodthirsty_rogue_to_alive_playerclan_leader",
-        "thoughts": [
-            "Tells {PRONOUN/m_c/self} {PRONOUN/m_c/subject} {VERB/m_c/are/is} going to kill r_c and take their position in c_n"
-        ],
-        "random_status_constraint": [
-            "leader",
-            "deputy",
-            "medicine cat"
-        ],
-        "main_trait_constraint": [
-            "bloodthirsty"
-        ],
-        "random_outside_status": [
-            "clancat"
-        ],
-        "random_living_status": [
-            "living"
-        ]
-    },
-    {
-        "id": "vengeful_rogue",
-        "thoughts": [
-            "Wants vengeance on {PRONOUN/m_c/poss} past Clan"
-        ],
-        "main_trait_constraint": [
-            "vengeful"
-        ]
-    },
-    {
-        "id": "strange_rogue",
-        "thoughts": [
-            "Crowns {PRONOUN/m_c/self} the ruler of all rogues"
-        ],
-        "main_trait_constraint": [
-            "strange"
-        ]
-    },
-    {
-        "id": "shameless_rogue",
-        "thoughts": [
-            "Is eating kittypet kibble",
-            "Stole some random kittypet's food",
-            "Is eating dog food {PRONOUN/m_c/subject} found",
-            "Is putting {PRONOUN/m_c/poss} own scent on c_n's clearly-marked borders"
-        ],
-        "main_trait_constraint": [
-            "shameless"
-        ]
-    },
-    {
-        "id": "cold_rogue",
+        "id": "general_rogue_thoughts",
         "thoughts": [
-            "Thinks about how {PRONOUN/m_c/subject} {VERB/m_c/don't/doesn't} need anyone else"
-        ],
-        "main_trait_constraint": [
-            "cold"
+            "Wishes the other rogues would leave {PRONOUN/m_c/object} alone",
+            "Snarls at any cat that comes near",
+            "Wanders off on {PRONOUN/m_c/poss} own",
+            "Has been going missing quite a lot lately",
+            "Sniffs for {PRONOUN/m_c/poss} next meal",
+            "Wonders if {PRONOUN/m_c/subject} can find a warm place to sleep tonight",
+            "Was chased by a fox recently",
+            "Wishes {PRONOUN/m_c/subject} {VERB/m_c/were/was} in a warm nest",
+            "Wonders what it's like to live with Twolegs",
+            "Is struggling to survive on {PRONOUN/m_c/poss} own",
+            "Is fighting with another rogue over food",
+            "Laughed at the thought of dead cats 'watching over' c_n",
+            "Debates if {PRONOUN/m_c/poss} previous Clan was fair in casting {PRONOUN/m_c/object} out",
```

Comment:
not as far as i know other than constraining to the "otherclan" set of backgrounds, but this thought was here since before my edit. i assume that the implication is that the rogue used to live in another (non-player) clan? 
i can remove it if that's what's wanted :0

Reply:
This was under "general rogue thoughts" so I think it's available for every rogue regardless of background - but yeah if you don't mind deleting it even if you didn't make it, thank you!